docs/source/torch.rst

```diff
@@ -263,11 +263,8 @@ Examples::
     :toctree: generated
     :nosignatures:
 
-    no_grad
```

Comment:
Hey @adamjstewart! I'll let @brianjo look at the Sphinx update, but why does this PR remove these entries?

Reply:
Sphinx raises a warning when entries in the index appear in multiple places in the docs. I believe this was always intended to be the case, but Sphinx 4 fixed a bug that prevented it from being detected properly. See the errors raised by previous commits. The error messages suggest adding `:noindex:` to one of the entries but that didn't seem to work for me. I'm guessing this has something to do with things being auto-generated.

TL;DR this is a hack to get the documentation tests to pass. I welcome any suggestions of alternative ways to get the tests to pass but ran out of ideas myself.

Reply:
Interesting; thanks for the explanation. Let's wait for our docs expert(s) to weigh-in.

Reply:
Is there a way to see the rendered documentation after this PR? I would expect the removed entries to still show up like they do on https://pytorch.org/docs/stable/torch.html#locally-disabling-gradient-computation as rows in the table below the examples.

Reply:
@seemethere Eli, is there a way for us to get the output of the docs build now that we're on actions?

Reply:
The rendered docs are available from the GithubBot post: https://github.com/pytorch/pytorch/pull/70309#issuecomment-999723231


https://docs-preview.pytorch.org/70309/index.html

Reply:
@adamjstewart your change removes number of entries from the doc, compare:
https://pytorch.org/docs/stable/torch.html#locally-disabling-gradient-computation
with
https://docs-preview.pytorch.org/70309/torch.html#locally-disabling-gradient-computation

`no_grad` is now missing in the preview

Reply:
Correct, see my comment above: https://github.com/pytorch/pytorch/pull/70309#discussion_r782322575

If anyone knows how to fix this without removing `no_grad` from the docs please let me know, but I'm out of ideas. The current docs are not valid as far as Sphinx is concerned, you aren't allowed to have the same entry in the index multiple times.

Reply:
I'm not sure, I understand the comment: why do you think `no_grad` index appears in multiple places? Greping thru [torch.rst](https://github.com/pytorch/pytorch/blob/master/docs/source/torch.rst) shows it mentioned only once in that .rst file, though I'm not at all a doc expert

Reply:
It's also mentioned in autograd.rst. All files are combined to create a single index for the documentation.

Reply:
@adamjstewart I can open it in incognito browser, so I always thought that they are public